### DISCUSSION TITLE
Monthly ESMValTool community meeting November 2023

### DISCUSSION THREAD

**Discussion by paultgriffiths** [ESMValGroup/Community#132] [Meetings]
The next online community meeting will be on 

November 15th at 2pm UTC / 3pm CET

The duration will be one hour.  [Join the Zoom meeting](https://ncas.zoom.us/j/98260572431?pwd=Vkh1K3c1Y25KcjVJY3NwT1NDWjVCQT09)

Calendar invitations to the meeting are sent to the ESMValTool mailing list, join the list [here.](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list)

**Format and agenda** 

Anyone with an interest in ESMValTool is welcome to join and ask questions

We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.

Using the input from the introduction round, we choose some topics to talk about in more detail.

If you already know that you would like to bring up a particular topic in the meeting, you can mention the GitHub issue on the topic in a comment below. That will make it easy to find for the other people in the meeting and we can already start discussing there before the meeting. Just create a new GitHub issue about your topic if there is no open issue yet.

GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the previous meeting are available in #125

**Comment by rswamina** [comment]
Dear @ESMValGroup/esmvaltool-developmentteam, 

We would like to invite abstracts to an upcoming session on Model Evaluation and Benchmarking at the EGU General Assembly 2024 in Vienna,  14-19 April 2024:

**ESSI2.6 :** **Benchmarking and Evaluation of Earth System Models**
Session Description:
Earth System Models (ESMs) have evolved considerably in complexity, capability and scale as evidenced in projects such as the Coupled Model Intercomparison Project Phase 6 and the forthcoming CMIP7 project. There is therefore a need to credibly assess such developments and capabilities for effective research on climate variability and change. The CMIP7 Model Benchmarking Task Team was established by the World Climate Research Programme to advance the full integration of routine benchmarking and evaluation into the CMIP workflow.  As part of this initiative, we invite studies that explore novel approaches for benchmarking and evaluation of ESMs including cross-domain and process -based evaluation, observational uncertainties, science and performance metrics and benchmarks. The development of tools and methods including Machine Learning and Artificial Intelligence approaches are also welcomed.
Abstract Submission and Session details link : [https://meetingorganizer.copernicus.org/EGU24/session/47985](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmeetingorganizer.copernicus.org%2FEGU24%2Fsession%2F47985&data=05%7C01%7Cr.swaminathan%40READING.AC.UK%7C4a135b1528e4407574f308dbe5cf6901%7C4ffa3bc4ecfc48c09080f5e43ff90e5f%7C0%7C0%7C638356451219432431%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=vfbhG0Y2gnpezCWpk%2BXgMYVQ3uca3GLG1H2oBswunFo%3D&reserved=0)
 Deadline for submission: **Wednesday, 10 January 2024, 13:00 CET**
 Deadline for submissions (applying for support):  **Friday, 1 December 2023, 13:00 CET**
Submission guidelines: [https://www.egu24.eu/programme/how_to_submit.html](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.egu24.eu%2Fprogramme%2Fhow_to_submit.html&data=05%7C01%7Cr.swaminathan%40READING.AC.UK%7C4a135b1528e4407574f308dbe5cf6901%7C4ffa3bc4ecfc48c09080f5e43ff90e5f%7C0%7C0%7C638356451219432431%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Pszs6gR11%2BpvYF7k%2Fn9BHWCHzA6vZqRQ9A%2FkkeT1bjI%3D&reserved=0)
 
Please feel free to share widely and contact the Session Convenors if you have any questions.

Regards,
Ranjini Swaminathan, Birgit Hassler, Forrest Hoffman, Alicia Hou and Joshua Dorrington

**Comment by paultgriffiths** [comment]
Updates:
- @malininae Has a relative humidity calculation.  Will open a pull request
- @malininae [working on a Fire Weather Index calculation](https://github.com/ESMValGroup/ESMValTool/issues/3412) but is using a lot of memory (185Gb for 20 yrs of data at 3hr and 0.25 degree resolution).  See https://github.com/ESMValGroup/ESMValCore/issues/1915. Share recipe as a way to see if or where Dask is not correctly configured.
- @rswamina EGU24 has a session (see above). 🚀 
- @egalytska Tutorial not accepted at EGU 😞 .  A student is very helpfully reviewing the [tutorial](https://tutorial.esmvaltool.org/)
- @glpotter WMO is setting up lead centers to host (ideally) CMOR-ised reanalysis data.  ESMValTool will be used for CMOR-ising and regridding.  Will keep under review 👀 
- @egalytska Use of [CVDP](https://github.com/NCAR/CVDP-ncl) in ESMValTool.  Issue of not giving output of tripole index for Interdecadal Pacific Oscillation https://github.com/ESMValGroup/ESMValTool/issues/3419.  https://github.com/NCAR/ncl/issues/64 is noted and may mean that it may not make much sense to put dev time into NCL which is EOL.  Suggestion to look for existing CVDP functionality that may already have been implemented, e.g. https://github.com/ESMValGroup/ESMValTool/pull/1560  🧠 
- @zklaus https://github.com/ESMValGroup/ESMValTool/discussions/3403 is coming up.  @zklaus is the release manager.  Who will be the next release manager (Feb 2024 and June 2024)?  Opportunities to join as deputy release manager and observe process.  NB [detailed notes](https://docs.esmvaltool.org/en/latest/community/release_strategy/index.html)  are available to guide the release manager, and the previous release manager stays on to help you. 🔍 
- @lukruh It would be a good idea to keep the CMOR-iser for old versions of reanalysis.  @bouweandela - good idea to support multiple versions.  Issue at https://github.com/ESMValGroup/ESMValTool/issues/3430. 👏 
- @malininae #133 location/date of next workshop - preference to tag onto the week before or after the next CMIP7 task team meeting in May. 🗺️ 📅
